STACpopulator/metadata_parsers.py

```diff
@@ -0,0 +1,61 @@
+import lxml.etree
+import requests
+
+
+def nc_attrs_from_ncml(url):
+    """Extract attributes from NcML file.
+
+    Parameters
+    ----------
+    url : str
+    Link to NcML service of THREDDS server for a dataset.
+
+    Returns
+    -------
+    dict
+    Global attribute values keyed by facet names, with variable attributes in `__variable__` nested dict, and
```

Comment:
I would re-word this a bit. It's not clear from this description that `__group__` and `__variable__` are dictionary keys

Reply:
Will do that when updating the docs. This is some original function from the earlier `star-ingest` project that I haven't cleaned up yet.